Formula/picard-tools.rb

```diff
@@ -0,0 +1,20 @@
+class PicardTools < Formula
+  desc "Tools for manipulating HTS data and formats"
+  homepage "https://broadinstitute.github.io/picard/"
+  url "https://github.com/broadinstitute/picard/releases/download/2.16.0/picard.jar"
+  sha256 "01cf3c930d2b4841960497491512d327bf669c1ed2e753152e1e651a27288c2d"
+
+  bottle :unneeded
+
+  depends_on :java
```

Comment:
The website says _“For the tools to run properly, you must have Java 1.8 installed”_. This needs to be reflected here.

Reply:
Do we know for a fact that it doesn't work with Java 9?

Reply:
It should have at least 1.8+